client/src/components/requestsTable/row.tsx

```diff
@@ -73,10 +73,21 @@ function Row(props:RowProps) {
               Accept
             </Button>
           ) : (
-            <CarAdminModel />
+            <Button
+              onClick={() => navigate(`check/${car.id}`)}
```

Comment:
onClick={() => navigate(`check/${car.id}`)}
why use navigate you can use 
```suggestion
             <Link to={`check/${car.id}`}> </Link>
```